client/components/Appointment.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import $ from 'jquery';
+import moment from 'moment';
+import LevelOne from './LevelOne.jsx';
+import LevelThree from './LevelThree.jsx';
+
+class Appointment extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      businessId: Math.floor(Math.random() * Math.floor(100)),
+      businessName: '',
+      businessMax: 0,
+      businessAddress: '',
+      businessCity: '',
+      businessState: '',
+      businessZip: '',
+      timeRange: [],
+      guestCount: '1',
+      selectedDate: '',
+      selectedTime: '',
+      firstName: '',
+      lastName: '',
+      email: '',
+      phone: '',
+      nextTwoWeeks: [],
+      view: 1,
+    };
+    this.next = this.next.bind(this);
+    this.back = this.back.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.makeAppointment = this.makeAppointment.bind(this);
+  }
+
+  componentDidMount() {
+    const defaultTimes = () => {
```

Comment:
I think you might want to define your defaultTimes function outside of componentDidMount, and then call it in there.  It would probably help access to it for testing.  Otherwise I guess it only matters if you want access to the function outside of that scope.